index.html

```diff
@@ -79,16 +81,29 @@ <h1 class="heading-primary">
             </li>
           </ul>
         </div>
-        <img src="imgs/arrow down.png" alt="Arrow down" class="navigator" />
-      </section>
+        <img
+          src="imgs/mobile-version/arrow-down.png"
+          alt="Arrow down"
+          class="navigator"
+        />
+      </div>
+    </header>
 
+    <!--------------------------->
+    <!-- MAIN PART OF THE PAGE -->
+    <!--------------------------->
+    <main class="bg-color main">
       <!----------------------->
       <!-- WORKS SECTION -->
       <!----------------------->
-      <section class="section-works">
+      <section class="section-works section section--lg">
         <div class="works-header">
           <h2 class="heading-secondary">My Recent <br />Works</h2>
```

Comment:
- According to [this requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md#:~:text=You%20must%20replicate%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20info%20from%20the%20Figma%20templates.), Kindly make sure to match the Figma design for the layout of this title on the desktop version

**Your Design**
![Screenshot from 2023-05-04 20-03-34](https://user-images.githubusercontent.com/97834160/236294530-b214784d-6204-42c3-9c76-55b04ceffae1.png)

**Figma Design**
![Screenshot from 2023-05-04 20-12-15](https://user-images.githubusercontent.com/97834160/236294549-9d52d157-e58b-4aed-8267-6e227206256f.png)